libfwupdplugin/fu-crc.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2022 Richard Hughes <richard@hughsie.com>
```

Comment:
2023 now

Reply:
I think this should be the date we wrote the code, not committed the code